### PR TITLE
Documaster API: draft 9

### PR DESCRIPTION
**Pull request by ivaylomitrev**
Changes in the API documentation:

* Support for access-group endpoints
* Support for explicit permissions on Classification and Section
* Support for read-only effective permissions on all resources
* Support for service permissions

### CODE REVIEW

File: v1/endpoints.md [hunk 1]
```diff
@@ -942,6 +948,213 @@ See [RFC 6266](http://tools.ietf.org/html/rfc6266) (description of the Content-D
...
+```
+
+- `data`
+  - contains the (potentially empty) resource with the specified ID
+  - at most 10 nested resources per type will be returned; more information on retrieving additional resources can be found in `__resources`
+
+Response codes:
+- `200 OK`
+- `400 Bad request`
+- `403 Unauthorized`
```
Comment (phillipj) [inline v1/endpoints.md:1041]: Haven't read every corner of the docs yet, so my following question might already be answered elsewhere in the docs as a general rule of thumb or similar..

Would trying to fetch a non-exising access group also result in 403?

Comment (ivaylomitrev) [reply v1/endpoints.md:1041]: Yes, it will :)

Comment (phillipj) [reply v1/endpoints.md:1041]: Cool 👍 Assuming that is to not leak data by disclosing what exists or not... I've been surprised by that kind of behaviour before being a client, is it described as a heads-up anywhere?

Comment (ivaylomitrev) [reply v1/endpoints.md:1041]: It somewhat is in the beginning of endpoints.md, but I agree with you that we can expand it to be more explicit.

As a rule of thumb with this API you will get 404 if you hit a non-existing endpoint and a 403 if you request a non-existing resource or one you do not have access to.

**Pull request merged**